README.md

```diff
@@ -2,7 +2,7 @@
 
 # Math Magicians
 
-**Math magicians project A class based calculator app built using react.**
+**Math magicians project A hooks based calculator app built using react.**
 
 ## Built With
 
```

Comment:
- [ ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

![00Mit](https://user-images.githubusercontent.com/79058364/182822618-fdfb2523-cb23-4060-9f72-75ec8c29e4cd.png)


#### The reason for this is that there is no `MIT.md` file in the root directory of this project. Kindly attach the file. 👍
-  Please, consider fixing the broken link to the MIT.md file 🥇